xonsh/ptk_shell/shell.py

```diff
@@ -195,7 +203,21 @@ def singleline(
                 except (AttributeError, TypeError, ValueError):
                     print_exception()
 
+        if env["ENABLE_ASYNC_PROMPT"]:
+            # once the prompt is done, update it in background as each future is completed
+            updators = [
```

Comment:
How about `updaters`?

```suggestion
            updaters = [
```